src/bosh_azure_cpi/spec/unit/azure_client/managed_disks_spec.rb

```diff
@@ -457,6 +457,40 @@
           azure_client.delete_managed_disk(resource_group, disk_name)
         end.not_to raise_error
       end
+
+      it 'should raise error' do
```

Comment:
what is the context of this case? better to add context, otherwise people will be confused when only reading test result.
same for some other places.